### DISCUSSION TITLE
How can I do profiling on sea-orm in a web app?

### DISCUSSION THREAD

**Discussion by lidh15** [SeaQL/sea-orm#1025] [Q&A]
The database related operations, mostly queries, are very slow, three times slower than the same tasks with python flask_sqlalchemy and I cannot figure it out, is there a profiling tool for axum+sea-orm?

**Comment by billy1624** [comment]
Hey @lidh15, welcome! First thing first, are you running the Rust application in release mode? `cargo run --release`

If you want to profile the query operations. Please check
- https://github.com/SeaQL/sea-orm/discussions/372
- https://github.com/SeaQL/sea-orm/pull/373
